static/style.css

```diff
@@ -86,38 +86,14 @@ td > div {
 	opacity: 2.0;
 }
 
-/*CSS styling for the bottom slider*/
```

Comment:
Since myslider2 has the same CSS code as myslider1, I removed myslider2 and changed references to it to myslider1.